modules/ROOT/pages/api-led-test.adoc

```diff
@@ -64,60 +63,61 @@ http://localhost:8081/api/greeting
 ----
 
 . Return to Studio and notice that there's a dashed line around *Set Payload*.
-. In the canvas, click *Set Payload* to open the *Set Payload* node in the *Mule Debugger* above the canvas. Open *Set Payload* and see that the payload value is `Hi.`, three characters.
-. Click the curved yellow arrow above the debugger window.
+. In the canvas, click *Set Payload* to open the *Set Payload* node in the *Mule Debugger* above the canvas. 
+. Open *Set Payload* and see that the payload value is `Hi.`, three characters.
```

Comment:
Do you need the three characters? Is it part of the UI? Should it be in parans or something?

Reply:
I think it's pointed out because the 3 characters is what causes the error.